src/libraries/System.Memory.Data/README.md

```diff
@@ -19,7 +19,7 @@ Console.WriteLine(data.ToString()); // prints "some data"
 
  To/From bytes:
 ```C# Snippet:BinaryDataToFromBytes
-byte[] bytes = Encoding.UTF8.GetBytes("some data");
+byte[] bytes = "some data"u8.ToArray();
```

Comment:
I think we have the ability to show off version specific examples. Might be worth asking @gewarren

Reply:
Unfortunately in the API ref docs, the same example would be shown for all versions. But this is in a README.md file, so I don't think it would affect docs anyway.